test/unit/dialog-renderer.spec.js

```diff
@@ -5,7 +5,7 @@ import {dialogOptions} from '../../src/dialog-options';
 import {configure} from '../../src/aurelia-dialog';
 import {TestElement} from '../fixtures/test-element';
 
-describe('the Dialog Renderer', () => {
+describe('the Dialog Renderer', function () {
```

Comment:
`describe` and `it` set the context - `this`, using a lambda prevents it. If someone tries to use `this` without noticing a lambda has been used, he(me :sweat_smile:) can loose quite some time.